lib/replace/js.js

```diff
@@ -51,7 +50,9 @@ const replaceJs = (code, espreeOptions) => {
         const raw = node.value.raw.replace(allRegex.templateSelectors, function (txt, p1, p2, p3) {
           // p3 contains the content of the class=' or id=", so let's replace them
           const newValue = ` ${p3} `;
-          const replacedAttr = newValue.replace(newValue, match => replaceString(match, regex, ' ', { countStats: false }));
+          const replacedAttr = p1 === 'class' ?
```

Comment:
What about following for readability:

```js
const selectorLib = p1 === 'class' ? selectorsLibrary.getClassSelector() : selectorsLibrary.getIdSelector();
const replacedAttr = newValue.replace(newValue, match => replaceString(match, selectorLib.getAll({ regex: true }), ' ', { countStats: false }));
```

Reply:
Sure, will do tonight.

Reply:
It's done. I'll now work on the name generator fix and then on `rename-css-selector` project to use the new API.

I was planning to change the format of the mapping file so it's hierarchical instead of a plain dictionnary: one per `BaseLibrary` instance

Instead of:
```
{
"more": "a",
[...]
}
```
have:
```
{
"keyframes": { "stop": "a", [...] },
"class": { "more": "a", [...] },
"id": { "id": "a", [...] },
"variables": { "red": "a", [...] }
}
```